sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -42,8 +42,18 @@ const Header = styled('header', Container, {
   width: '100%',
 
   '@md': {
+    px: 0,
```

Comment:
Could you please explain the reason of this change?

Reply:
I think this was from where I was setting the maxWidth at each media query, but didn't take into account the padding that is already there as part of the container. Will remove this and fixed maxWidth sizes 👍